e2e/worker/test.sh

```diff
@@ -14,7 +14,8 @@ exit_with_message() {
 }
 
 cleanup() {
-    rm -f evil.ts included.ts
+    rm -f evil.ts
+    find . -type f -name '_addendum_*.ts' -exec rm {} \;
```

Comment:
Can we use `git clean`? Then this method doesn't need all the generated files listed in it...

Reply:
I am afraid that it might be destructive since you could have some local changes and invoke test.sh then it would take back local changes which isn't something we desire. so I wanted to be explicit about what's been generated and what's not.